app/components/NestedCheckboxes.tsx

```diff
@@ -70,20 +76,25 @@ function NestedCheckboxNode({
         label={
           <>
             <span className="padding-right-1">{topLevelNodeProps.label}</span>
-            {link && (
-              <>
-                (
-                <Link
-                  to={link}
-                  onClick={(e) => {
-                    e.stopPropagation()
-                  }}
-                >
-                  Details
-                </Link>
-                )
-              </>
-            )}
+            {link &&
+              (isExternalLink ? (
```

Comment:
Change the condition in the ternary operator to isExternalLink(link) to avoid checking if link is defined twice.